specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -440,7 +440,7 @@ For example,
 
 . A Deployer may have mapped a security role to
 a user group in the operational environment or may depend on the default
-mapping of security roles to user groups as defined by the Java EE
+mapping of security roles to user groups as defined by the Jakarta EE
 Security API specification. In this case, the user group of the calling
```

Comment:
I think this specification may now be called Jakarta Security

Reply:
Right